### DISCUSSION TITLE
Add entry to history / back stack

### DISCUSSION THREAD

**Discussion by tmbenhura** [unpoly/unpoly#320] [Q&A]
Is there a way to insert an entry in the history / back stack? 

For example, on a mobile device, you click a link via email to view an order. When you hit back, the browser would take you to the orders listing instead of closing the browser.

**Comment by triskweline** [comment]
This would require use of internal API:

```js
let location = up.history.location
up.history.replace('/order-list')
up.history.push(location)
```

The internal part here is `up.history.replace()`, so no guarantees for future versions.

**Comment by tmbenhura** [comment]
Noted with thanks.
